opm/output/eclipse/Summary.cpp

```diff
@@ -748,6 +748,9 @@ inline quantity glir( const fn_args& args ) {
 
         const auto& production = well->productionControls(args.st);
         if (! has_vfp_table(sched_state, production.vfp_table_number)) {
+            const double eff_fac = efac(args.eff_factors, well->name());
+            alq_rate += eff_fac * xwPos->second.rates.get(rt::alq, production.alq_value);
```

Comment:
Are you sure you want to **add** the values here?  Please keep in mind that function `glir()` is the backend for the field level vector [`FGLIR`](https://github.com/OPM/opm-common/blob/010397cda334d6901ad7b42c95fd35e2b5e613b2/opm/output/eclipse/Summary.cpp#L2476), the group level vector [`GGLIR`](https://github.com/OPM/opm-common/blob/010397cda334d6901ad7b42c95fd35e2b5e613b2/opm/output/eclipse/Summary.cpp#L2280), and the well level vector [`WGLIR`](https://github.com/OPM/opm-common/blob/010397cda334d6901ad7b42c95fd35e2b5e613b2/opm/output/eclipse/Summary.cpp#L2141).  The only difference between these cases is the contents of [`args.schedule_wells`](https://github.com/OPM/opm-common/blob/010397cda334d6901ad7b42c95fd35e2b5e613b2/opm/output/eclipse/Summary.cpp#L737).  Might it make sense to have special case handling of `WGLIR` for this purpose?

Reply:
I think it makes sense to add, but instead of 'return' above it should be 'continue' (with rates converted according to unit).